pillar/kibana.sls

```diff
@@ -258,7 +254,7 @@ elasticsearch:
             <!subteam^S9PK3B39V|devopseng> The upstream service on {0} is not responding to Nginx
           alert_text_args:
             - minion_id
-          slack_webhook_url: {{ slack_webhook_url_odl }}
+          slack_webhook_url: __vault__::secret-operations/global/slack-odl/slack_webhook_url>data>value
```

Comment:
This feels like a good case of setting the `slack_webhook_url` at the beginning of the file and referencing it throughout.